### PR TITLE
Feature/base file structure and styles

### PR DESCRIPTION
**Pull request by Dmytro1918**
Font was added. Structured files iaw SMACSS.

### CODE REVIEW

File: src/styles/base/variable.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//Colors of first page
+$background-color:#F3F2EA;
```
Comment (tihon-veliar) [inline src/styles/base/variable.scss]: Please, rename this color name. 
We don't have a base color of the bg for the project.
Please, name all colors by their name.

File: src/styles/base/variable.scss
Comment (tihon-veliar) [inline src/styles/base/variable.scss:1]: We need to use a css variable, not the scss. 
Scss var doesn't work for the styled component.

File: src/styles/base/variable.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+$purple:#604689;
+$lollipop-purple:#6F28DE;
+
+// font sizes - first page
+$title-font-size:180px;
+$description-font-size:24px;
+$links-font-size:20px;
+
+// font sizes - second page
+$first-title-fontsize:258px;
```
Comment (tihon-veliar) [inline src/styles/base/variable.scss]: Font sizes will be flexible, so we don't need vars for them.

Also, we have the topographic file, so all things about fonts should be there

File: src/styles/base/reset.css
Comment (tihon-veliar) [inline src/styles/base/reset.css:1]: Please, find a reset code for this project.

File: src/styles/styles.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import 'base/fonts';
+@import 'base/mixins';
+@import 'base/reset';
```
Comment (tihon-veliar) [inline src/styles/styles.scss]: import of the reset must be in the first place.

File: src/styles/styles.scss
Comment (tihon-veliar) [inline src/styles/styles.scss:1]: This file is better to name as main.scss or index.scss

**Review by tihon-veliar — CHANGES_REQUESTED** [review]
Please, take a look at my comments.

**Review by Dmytro1918 — COMMENTED** [review]
Changes were applied.

**Review by tihon-veliar — APPROVED** [review]

**Pull request merged**